wren-ui/migrations/20240125081244_create_model_column_table.js

```diff
@@ -7,6 +7,7 @@ exports.up = function (knex) {
     .createTable('model_column', (table) => {
       table.increments('id').comment('ID');
       table.integer('model_id').comment('Reference to model ID');
+      table.foreign('model_id').references('model.id').onDelete('CASCADE');
```

Comment:
@andreashimin can you put this to a new migration file ? I think migration file will not rerun if content change.

Conceptually, I think we should treat migration files as immutable files.

Reply:
I have made new migration for adding foreign key according to the conclusion.